x/emissions/module/rewards/topic_rewards.go

```diff
@@ -130,90 +83,85 @@ func PickChurnableActiveTopics(
 			Logger(ctx).Debug("Error getting topic: ", err)
 			continue
 		}
-		// Loop over and run epochs on topics whose inferences are demanded enough to be served
-		// Check the cadence of inferences, and just in case also check multiples of epoch lengths
-		// to avoid potential situations where the block is missed
-		if k.CheckWorkerOpenCadence(block, topic) {
-			ctx.Logger().Debug(fmt.Sprintf("ABCI EndBlocker: Worker open cadence met for topic: %v metadata: %s . \n",
-				topic.Id,
-				topic.Metadata))
 
-			// Update the last inference ran
-			err = k.UpdateTopicEpochLastEnded(ctx, topic.Id, block)
-			if err != nil {
-				ctx.Logger().Warn(fmt.Sprintf("Error updating last inference ran: %s", err.Error()))
-				continue
-			}
-			// Add Worker Nonces
-			nextNonce := types.Nonce{BlockHeight: block}
-			err = k.AddWorkerNonce(ctx, topic.Id, &nextNonce)
-			if err != nil {
-				ctx.Logger().Warn(fmt.Sprintf("Error adding worker nonce: %s", err.Error()))
-				continue
-			}
-			ctx.Logger().Debug(fmt.Sprintf("Added worker nonce for topic %d: %v \n", topic.Id, nextNonce.BlockHeight))
+		ctx.Logger().Debug(fmt.Sprintf("ABCI EndBlocker: Worker open cadence met for topic: %v metadata: %s . \n",
```

Comment:
This reference to ABCI Endblocker is no longer valid since this code has been moved to topic_rewards